src/pages/[platform]/build-a-backend/push-notifications/push-notifications-migration-guide/index.mdx

```diff
@@ -0,0 +1,160 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Migrate from v5 to v6',
+  description: 'Learn more about the migration steps to upgrade to v6',
```

Comment:
description: 'Learn more about the migration steps to upgrade Push Notifications APIs for Amplify JavaScript v5 to v6',